tasks/fix-cat2.yml

```
@@ -1,25 +1,21 @@
 ---
-- name: "MEDIUM | RHEL-07-010030 | PATCH | The Red Hat Enterprise Linux operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a graphical user logon."
-  command: "true"
-  changed_when: no
+### RHEL-07-010030 | RHEL-07-010040 combined as related tasks in regards to a config file no other content will be in.
+- name: "MEDIUM | RHEL-07-010030 | RHEL-07-010040 | PATCH | The Red Hat Enterprise Linux operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a graphical user logon."
+  template:
+      src: 01-banner-message.j2
```

avoid lots of tiny files; convert literal newlines to "\n"

```suggestion
      content: |
          [org/gnome/login-screen] 
          banner-message-enable=true
          banner-message-text='{{ rhel7stig_logon_banner | replace(newline, "\n") }}'
```

---

ack on leaving this as `template`.  You still need to replace newlines in the banner with a literal `\n`